.isort.cfg

```diff
@@ -7,4 +7,4 @@ combine_as_imports=True
 use_parentheses=True
 line_length=88
 known_first_party=oca_github_bot
-known_third_party = aiohttp,appdirs,celery,gidgethub,github3,odoorpc,pytest,requests,setuptools
+known_third_party = aiohttp,appdirs,bs4,celery,gidgethub,github3,odoorpc,pytest,requests,setuptools
```

Comment:
I think is better to use the official package name (beautifulsoup4) instead of that dummy one. See https://pypi.org/project/bs4/. Don't you think?

Reply:
Ok, updated `setup.py` file and `requirements.txt` file accordingly. But this `isort.cfg` file I cannot change, because is filled automatically by `seed-isort-config` check, that interprets the `bs4` module like a package.